pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -120,36 +120,50 @@ in mkDerivation rec {
     ++ lib.optional withWebKit qtwebkit
     ++ pythonBuildInputs;
 
-  nativeBuildInputs = [ makeWrapper wrapGAppsHook cmake flex bison ninja ];
+  nativeBuildInputs = [ makeWrapper wrapQtAppsHook cmake flex bison ninja ];
 
   patches = [
     (substituteAll {
       src = ./set-pyqt-package-dirs.patch;
       pyQt5PackageDir = "${py.pkgs.pyqt5}/${py.pkgs.python.sitePackages}";
       qsciPackageDir = "${py.pkgs.qscintilla-qt5}/${py.pkgs.python.sitePackages}";
     })
+    ./fix-qsci.patch
   ];
 
+  #QT_QPA_PLATFORM_PLUGIN_PATH="${qt5.qtbase.bin}/lib/qt-${qt5.qtbase.version}/plugins/platforms";
+  QT_QPA_PLATFORM_PLUGIN_PATH="/nix/store/9j0acz9qqp1lygwif5jncpz8hsyfmylw-qtbase-5.15.9-bin/lib/qt-5.15.9/plugins/platforms";
```

Comment:
You can't hardcode path like this. The has will change when qtbase dependency or qtbase itself will change. You can use `${qtbase.bin}/lib/qt-${qtbase.version}/plugins/platforms` instead.

Reply:
This was an experiment and the whole line should have been removed.